scripts/phala-subbridge/khala-e2e.config.json

```diff
@@ -0,0 +1,90 @@
+{
+  "relaychain": {
+    "bin": "./bin/polkadot",
+    "chain": "rococo-local",
+    "nodes": [
+      {
+        "name": "alice",
+        "wsPort": 19944,
+        "port": 10333,
+        "nodeKey": "807e26fb3fd3d694b8b5f78a04d4bcff2e94a6f2e87bc809d9690e771bba8aca",
+        "basePath": "data/relay_alice",
+        "flags": [
+          "--rpc-cors=all"
+        ]
+      },
+      {
+        "name": "bob",
+        "wsPort": 19244,
+        "port": 12333,
+        "basePath": "data/relay_bob"
+      },
+      {
+        "name": "charlie",
+        "wsPort": 19344,
+        "port": 13333,
+        "basePath": "data/relay_charlie"
+      },
+      {
+        "name": "dave",
+        "wsPort": 19444,
+        "port": 14333,
+        "basePath": "data/relay_dave"
+      }
+    ],
+    "genesis": {
+      "runtime": {
+        "runtime_genesis_config": {
+          "configuration": {
+            "config": {
+              "validation_upgrade_frequency": 10,
+              "validation_upgrade_delay": 10
+            }
+          }
+        }
+      }
+    }
+  },
+  "parachains": [
+    {
+      "bin": "./bin/khala-node",
+      "chain": "thala-dev-2004",
+      "balance": "1000000000000000000000",
+      "nodes": [
+        {
+          "name": "alice",
+          "wsPort": 9944,
+          "port": 30333,
+          "nodeKey": "81bbf2ee046d4f57ab9ce02e822ac2a3192e4a7b38af757383f3dcb039c15e05",
```

Comment:
We may not need `nodeKey` here, I got an error yesterday when I do some testing on other stuff, not sure if you have encountered

Reply:
can you elaborate more on the error you saw please? I haven't seen any error regarding the node key so far

Reply:
just keep the conversion align, I will keep the nodekey for now until I encounter issue with it